t/04-check_vars_docu.t

```diff
@@ -44,6 +44,8 @@ my $error_found = 0;
 # ignore errors for now
 my $ignore_errors = 1;
 
+my $do_write = (defined $ARGV[$#ARGV] && $ARGV[$#ARGV] eq '--write') ? 1 : 0;
```

Comment:
tests aren't programs. Let it create a different filename, so it appears in git status - for those that want to check what's different

Reply:
done.